ios/FluentUI/Vnext/List/ListCell.swift

```diff
@@ -380,12 +383,20 @@ struct MSFListCellView: View, ConfigurableTokenizedControl {
             let horizontalCellPadding: CGFloat = tokens.horizontalCellPadding
             let leadingViewAreaSize: CGFloat = tokens.leadingViewAreaSize
 
-            Button(action: state.onTapAction ?? {
+            Button(action: {
                 if hasChildren {
                     withAnimation {
                         state.isExpanded.toggle()
                     }
                 }
+
+                var handledSelection = false
```

Comment:
Can you do this to avoid the var?

```
                guard let selectionAction = state.selectionAction, selectionAction(state) else {
                      onTapAction?()
                }
```